### PR TITLE
Redesigning homepage

### PR DESCRIPTION
**Pull request by Aliiiu**
# Redesigning IQ Wiki Homepage

closes https://github.com/EveripediaNetwork/issues/issues/1687

## Notes or observations

<br />
image

image

image

image

image

image

### CODE REVIEW

File: src/components/Landing/AboutIqgpt.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+          <Box color={'eventTextColor'} pb={'20px'}>
+            <Text fontSize={{ base: '14px', md: '16px' }}>
+              {`${t('aboutIQGPTOneBody')}`}
+            </Text>
+            <Text fontSize={{ base: '14px', md: '16px' }}>
+              {`${t('aboutIQGPTTwoBody')}`}
+            </Text>
+          </Box>
+          <HStack gap={{ base: 2, xl: 6 }}>
+            <Button
```
Comment (Adebesin-Cell) [inline src/components/Landing/AboutIqgpt.tsx:101]: You can use ``LinkButton`` here

File: src/components/Landing/AboutIqgpt.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+              href={'https://iqgpt.com/'}
+              target="_blank"
+              size="lg"
+              variant="solid"
+              px={{ base: 3, md: 5 }}
+              fontSize={'12px'}
+            >
+              Explore IQ GPT
+            </Button>
+            <Button
```
Comment (Adebesin-Cell) [inline src/components/Landing/AboutIqgpt.tsx:112]: ``LinkButton`` here too, to allow for consistent link buttons across the page.

**Comment by Adebesin-Cell** [comment]
One thing, try to reduce the size of the GIF of the robot, rn it's about 2.5 MB large. You can make it around 200kb - 500kb without losing the quality.

**Comment by Adebesin-Cell** [comment]
Screen Shot 2023-08-22 at 10 12 22 AM

Not sure if it's related to the PR, but that card in the carousel shrinks, unlike the others. Try to fix it.

**Review by Adebesin-Cell — CHANGES_REQUESTED** [review]

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
- this is the value for shadows on dark mode including the shadows on hover for categories:
border-radius: 8px;
background: var(--gray-700, #2D3748);
box-shadow: 0px 25px 50px -12px rgba(16, 16, 17, 0.25);

- re-inspect the IQ gpt section. the implementation is not right yet for md/sm breakpoints. and the gif is not visible.
- there's something weird about the featured wikis in both light and dark mode. they should look like this: 
image

**Comment by Adebesin-Cell** [comment]
@Aliiiu One thing, I think you should switch the order of the new banner and categories. On prod, the leaderboard doesn't show yet

**Comment by Adebesin-Cell** [comment]
@Aliiiu This PR is getting too large and long. It's been open for two weeks, pls try to round off and make smaller PRs for easy review and feedbacks.
Rn, It's messy to know what's going on in the PR.

**Review by Simplypheyie96 — APPROVED** [review]
I think we can merge now.
@Softdev1

**Comment by Aliiiu** [comment]
> I think we can merge now. @Softdev1

@Softdev1 I will have to resolve the PR

**Comment by Softdev1** [comment]
> > I think we can merge now. @Softdev1
> 
> @Softdev1 I will have to resolve the PR

No problem then @Aliiiu

**Pull request merged**